packages/perseus-editor/src/widgets/group-editor.tsx

```diff
@@ -50,19 +38,12 @@ class GroupEditor extends React.Component<Props> {
     };
 
     serialize: () => any = () => {
-        return _.extend({}, this.editor.current?.serialize(), {
-            metadata: this.props.metadata,
-        });
+        return _.extend({}, this.editor.current?.serialize());
```

Comment:
Could this be simplified?

```suggestion
        return this.editor.current?.serialize();
```

Reply:
Yeah, I was being cautious by not messing with it. Given that it's making a copy and I don't trust the editors not to mutate objects, I'd prefer:

```ts
return {...this.editor.current?.serialize()};
```